CONTRIBUTING.md

```diff
@@ -18,8 +25,64 @@ A post-install action is thrown. This action first install dependencies with `le
 
 At this point you are ready to contribute.
 
+## Storybook
+
+This tool will help you display component in a page where it will be isolated (rather than integrated in a page with many components).
+
+We are using the state of the art tool: [Storybook](https://storybook.js.org/).
+
+To launch it, you can run:
+
+```sh
+npm run storybook
+```
+
+## Keeping your forked repository up-to-date
+
+Before you start developping, it is important to always keep your forked repository **up-to-date**.
+
+### Link your forked repository to the original project
+
+For this, you will add a remote which points to the original repository (this has to be done only once):
+
+```sh
+git remote add upstream https://github.com/AxaGuilDEv/react-toolkit
+```
+
+### Update your master branch
+
+Then, each time you want to update your repository with the last changes, run:
+
+```sh
+git checkout master
+git fetch upstream
+git rebase upstream/master
```

Comment:
La doc dit de faire un merge, utile de le préciser ici ?

Reply:
Both merge and rebase are working, what's your issue with rebase?

https://medium.com/@topspinj/how-to-git-rebase-into-a-forked-repo-c9f05e821c8a

Reply:
Les 2 fonctionnent mais la doc officielle GitHub dit de faire un merge. Peut importe c'est bien comme ça.

Reply:
What's wrong with a `rebase`? Both work totally fine and should be the same if you don't work in master.

And rebase will be "cleaner" if you actually worked on master #TeamRebase

https://medium.com/@topspinj/how-to-git-rebase-into-a-forked-repo-c9f05e821c8a.

Reply:
This only thing: if, by mistake you started to work on your master branch, rebase will be easier to understand in the history.

Other issue I faced today: I worked on 2 different branches, my first branch (this one) is taking more time to be reviewed than my second branch (brandUlr). My second branch was merged into upstream repo before my first branch. Without rebase, it was a total mess in the history :)

Reply:
I also prefer to rebase, I would keep it like that